rasa_core_sdk/events.py

```diff
@@ -126,3 +126,28 @@ def AgentUttered(text=None, data=None, timestamp=None):
         "data": data,
         "timestamp": timestamp,
     }
+
+
+# noinspection PyPep8Naming
+def StartForm(form_name, timestamp=None):
+    return {
+        "event": 'start_form',
+        "form_name": form_name,
+        "timestamp": timestamp
+    }
+
+
+# noinspection PyPep8Naming
+def EndForm(timestamp=None):
+    return {
+        "event": "end_form",
+        "timestamp": timestamp
+    }
+
+
+# noinspection PyPep8Naming
+def FormListen(timestamp=None):
```

Comment:
what do we need this for?

Reply:
see https://github.com/RasaHQ/rasa_core/pull/878#discussion_r210857097

Reply:
I am not sure we added this yet, but can we not use a followup action? forcing the bot to run action listen as the next action?

Reply:
it hasnt been added yet, i use the follow up action in core but not on the SDK